extensions/typescript-language-features/yarn.lock

```diff
@@ -31,3 +31,8 @@ vscode-nls@^5.0.0:
   version "5.0.0"
   resolved "https://registry.yarnpkg.com/vscode-nls/-/vscode-nls-5.0.0.tgz#99f0da0bd9ea7cda44e565a74c54b1f2bc257840"
   integrity sha512-u0Lw+IYlgbEJFF6/qAqG2d1jQmJl0eyAGJHoAJqr2HT4M2BNuQYSEiSE75f52pXHSJm8AlTjnLLbBFPrdz2hpA==
+
+vscode-uri@^3.0.3:
```

Comment:
Please revert the yarn.lock changes as they cause our CI checks to fail

Reply:
Interesting, whats the point of that workflow?? I reverted the change, but now other actions are failing because of `--frozen-lockfile`